SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -150,7 +154,7 @@ void USpatialClassInfoManager::CreateClassInfoForClass(UClass* Class)
 
 	ForAllSchemaComponentTypes([&](ESchemaComponentType Type)
 	{
-		Worker_ComponentId ComponentId = SchemaDatabase->ClassPathToSchema[Class->GetPathName()].SchemaComponents[Type];
+		Worker_ComponentId ComponentId = SchemaDatabase->ClassPathToSchema[ClassPath].SchemaComponents[Type];
```

Comment:
If we have to make sure that ClassPath is NetworkRemapped before accessing, should we add an accessor function into ClassPathToSchema to ensure to user always does this?

Reply:
`ClassPathToSchema` only used in `IsSupportedClass` and `CreateClassInfoForClass`` so I think it's just remapping inside `CreateClassInfoForClass`. `

Reply:
Seconded. I'd like to abstract away the map/array interfaces we have like this.

Reply:
`ClassPathToSchema` is not externally facing for anything else in the GDK but this class. I think it's ok if it's used like this.